### DISCUSSION TITLE
有没有纯净版的vben？

### DISCUSSION THREAD

**Discussion by dshuais** [vbenjs/vue-vben-admin#4021] [General]
### Version

Vben Admin Pro

### Description

首先vben很棒！这是毋庸置疑的

但是能否出一个纯净版一点呢，因为当前版本太过庞大，用在中后台项目中有点大材小用。

### Proposed Solution

能否出一个纯净版的vben，看了源码 组件嵌套太深是不可避免的，能否考虑把通用内容构建为npm包的形式，让用户只专注于构建页面。

期待回复！

### Validations

- [X] Read the [docs](https://anncwb.github.io/vue-vben-admin-doc/)
- [X] Ensure the code is up to date. (Some issues have been fixed in the latest version)
- [X] I have searched the [existing issues](https://github.com/vbenjs/vue-vben-admin/issues) and checked that my issue does not duplicate any existing issues.

**Comment by jinmao88** [comment]
现在还不够纯净？

**Comment by likui628** [comment]
> 现在还不够纯净？

我大概理解他的意思，就是单项目，其它的依赖都是npm库。
![image](https://github.com/user-attachments/assets/20b8f3ec-b4b6-448a-8b04-3c6cc4c5f4ca)

**Comment by dshuais** [comment]
yes，我就是这个意思

**Reply by forose** [reply]
好像只能自己手动改了吧

**Comment by jinmao88** [comment]
单仓模式 应该不会有

**Comment by doraemonxxx** [comment]
Vben Pro, which already implemented the monorepo architecture in Vben 3, is an excellent architectural concept for building scalable web applications suitable for Small and Medium-sized Enterprises (SMEs). Although the code may seem overwhelming at first, you'll become familiar with it over time.

**Comment by AnLiBoBo** [comment]
我觉得可以出个组件库,类似于antd的pro component那样的

**Comment by CaoTianxing** [comment]
我抽出来了一个纯净版

**Reply by fantasysound** [reply]
大神，跪求一份纯净版的，你有联系方式吗？我加你，我的QQ号：88347748

**Reply by dshuais** [reply]
> 大神，跪求一份纯净版的，你有联系方式吗？我加你，我的QQ号：88347748

在 @CaoTianxing 的主页仓库内可以找到。

**Reply by CaoTianxing** [reply]
是的， 在我主页里面有，后续会继续更新缩减项目体积

**Comment by fantasysound** [comment]
非常感谢，好人一生平安




╭辉笑流╯&nbsp;
***@***.***

**Comment by cvpv** [comment]
换别的库吧，pure admin，我一直用这个，这个是单仓的。vben我还在研究

**Reply by pikepengfei** [reply]
后端对应的项目有么单仓的

**Reply by CaoTianxing** [reply]
mock的后端在我主页可以看得到，生产环境的后端目前没有
